Dockerfile

```diff
@@ -40,6 +40,7 @@ COPY --from=build-python /usr/local/lib/python3.11/site-packages /usr/local/lib/
 COPY --from=build-python /usr/local/bin /usr/local/bin
 
 COPY api /app/
+COPY .release-please-manifest.json /app/.release-please-manifest.json
```

Comment:
Just wondering whether, for aesthetics this should be: 

```
COPY .release-please-manifest.json /app/.versions.json
```

Something just feels a bit ick using release-please-manifest in the actual code.

Reply:
I know what you mean, but I think its better than grabbing the tag from Github. It's also the only place we are storing the individual package versions. Will change the filename